src/main/resources/static/static/js/updateOrder.js

```diff
@@ -0,0 +1,70 @@
+//计算商品总价
+function updateFinalPrice() {
+    var tr = $("#itemTable").find("tr[id]");
+    var summer = 0;
```

Comment:
please do not use `tr` which was mixed with HTML tag name, change to something like `cartTableElem`.